handle_http.go

```diff
@@ -305,10 +325,29 @@ func startDownloadWorker(source string, destination string, token string, transf
 			log.Debugln("Constructed URL:", transfer.Url.String())
 			if downloaded, err = DownloadHTTP(transfer, finalDest, token); err != nil {
 				log.Debugln("Failed to download:", err)
-				toAccum := errors.New("Failed to download from " + transfer.Url.String() +
-					" + proxy=" + strconv.FormatBool(transfer.Proxy) +
-					": " + err.Error())
-				AddError(toAccum)
+				var ope *net.OpError
+				var cse *ConnectionSetupError
+				errorString := "Failed to download from " + transfer.Url.Hostname() + ":" +
+					transfer.Url.Port() + " "
+				if errors.As(err, &ope) && ope.Op == "proxyconnect" {
+					log.Debugln(ope);
+					AddrString, _ := os.LookupEnv("http_proxy")
```

Comment:
What if you're using HTTPS? Should you then look for `https_proxy`?

Reply:
The current code manually disables proxies when HTTPS is in use.  So, https_proxy setting should be irrelevant for the client.

Reply:
Got it.